### PR TITLE
docs/common-coreboot-docs/dumping_logs.md: extend the document

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

File: docs/common-coreboot-docs/dumping_logs.md [hunk 1]
```diff
@@ -7,20 +7,27 @@ a running system.
...
+## `cbmem` utility
 
 When already migrated to Dasharo, it is possible to retrieve firmware logs from
 coreboot on a running system. A utility called `cbmem` can be used for that
 purpose. By obtaining the logs Dasharo team will be able to locate any issues
 with the firmware. This method requires Secure Boot to be disabled.
 
+Options to get `cbmem` utility:
+
+1. Download precompiled utility from [3mdeb cloud](https://cloud.3mdeb.com/index.php/s/zTqkJQdNtJDo5Nd/download)
```
Comment (sulewskiprzemyslaw) [inline docs/common-coreboot-docs/dumping_logs.md:26]: Under this link there's not only `cbmem` but also `flashrom` and binary for one of Dasharo products with wrong-signed keys. In my opinion we should change it in order not to cause any confusion.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/common-coreboot-docs/dumping_logs.md (hunk 1 above)
Comment (miczyg1) [reply docs/common-coreboot-docs/dumping_logs.md:26]: @sulewskiprzemyslaw that's the link you use in your test procedures throughout the whole docs repo. I think we should merge it as it is and then fix all occurrences of the cloud share with a separate share for cbmem only.

Comment (sulewskiprzemyslaw) [reply docs/common-coreboot-docs/dumping_logs.md:26]: @miczyg1 okay, I merge this PR and fix in another PR inconsistency with downloading `cbmem`

File: docs/common-coreboot-docs/dumping_logs.md [hunk 2]
```diff
@@ -7,20 +7,27 @@ a running system.
 
 ## System information
 
-One may use [Dasharo Tools Suite HCL report](../../dasharo-tools-suite/documentation#hcl-report)
+One may use [Dasharo Tools Suite HCL report](../dasharo-tools-suite/documentation#hcl-report)
```
Comment (sulewskiprzemyslaw) [inline docs/common-coreboot-docs/dumping_logs.md:10]: @miczyg1 I have a problem with this link. I just built the doc locally and this one redirects me to the empty site

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/common-coreboot-docs/dumping_logs.md (hunk 2 above)
Comment (miczyg1) [reply docs/common-coreboot-docs/dumping_logs.md]: Thanks, there were two broken DTS links in this file actually. Both fixed @sulewskiprzemyslaw

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**